test/node/search.js

```diff
@@ -39,37 +31,42 @@ describe('Search Tests', () => {
         await client2.logout();
     });
 
+    afterEach(async () => await helper.sleep(500));// wait to allow time for searches to return
+
     it('function: startBasicSearch [conversation], with event: searchStatus', async () => {
         const res = await Promise.all([
             client.startBasicSearch(conversation.topic),
             helper.expectEvents(client, [{
-                type: 'searchStatus'
+                type: 'searchStatus',
```

Comment:
Sorry I think I misunderstood. I made the changes you suggested. the very last rest I put into a multi line function because I didn't want to mix async/await with .then promises. I can make the very last predicate test one line if you want me to mix them but as is it makes one api call in the predicate.